mmv1/products/datacatalog/terraform.yaml

```diff
@@ -208,7 +208,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "basic_taxonomy"
         primary_resource_name: "fmt.Sprintf(\"tf_test_my_taxonomy%s\", context[\"random_suffix\"])"
```

Comment:
there is an issue here.. 

it needs `projects/{project}/locations/{location}/taxonomies/{taxonomies}` to import where taxonomies is a generated uuid, any way to circumvent this?

Reply:
@melinath any guidance here?

Reply:
I'll need to look into this closer; could you convert this to a non-draft PR so that it'll show up in my review queue?

Reply:
done, thank you